staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -291,10 +291,56 @@ type CSIDriverSpec struct {
 	// "csi.storage.k8s.io/pod.name": pod.Name
 	// "csi.storage.k8s.io/pod.namespace": pod.Namespace
 	// "csi.storage.k8s.io/pod.uid": string(pod.UID)
+	// "csi.storage.k8s.io/ephemeral": "true" iff the volume is an ephemeral inline volume
+	//                                 defined by a CSIVolumeSource, otherwise "false"
+	//
+	// "csi.storage.k8s.io/ephemeral" is a new feature in Kubernetes 1.16. It is only
+	// required for drivers which support both modes. Other drivers can leave pod info
```

Comment:
both modes => both "Persistent" and "Ephemeral" CSIVolumeModes.

Reply:
also make sure comments in both types.go files are in sync

Reply:
Done.